event_routing_backends/management/commands/tests/test_transform_tracking_logs.py

```diff
@@ -198,6 +217,12 @@ def test_transform_command(command_opts, mock_common_calls, caplog, capsys):
     expected_results = command_opts.pop("expected_results")
     transform_tracking_logs.CHUNK_SIZE = command_opts.pop("chunk_size", 1024*1024*2)
 
+    # Mock RouterConfiguration to return specific URLs
```

Comment:
Do we really need to update this test? Given if have `if router_urls:` in `prepare_to_send` method.